src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,76 @@
+.contacts {
+  @include pageGrid;
+
+  &__form {
+    grid-column: 1 / -1;
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    margin-bottom: 32px;
+
+    @include onDesktop {
+      grid-column: 1 / 6;
+    }
+  }
+
+  &__content{
+    padding: 14px 16px;
+    background: #F8F8FA;
+    border: 1px solid transparent;
+    border-radius: 8px;
```

Comment:
Consider using a variable for the border color to maintain consistency across the project.